### PR TITLE
Update contributing, getting started, and diffexp tutorial docs

### PR DESCRIPTION
**Pull request by tristanbrown**
@jvrakor @jkokosar

### CODE REVIEW

File: docs/tutorial-diffexp.rst [hunk 1]
```diff
@@ -165,31 +164,35 @@ usually used for time- or dosage- series.
...
 	:align: center
 
-	Relation schema showing a compare relation and three group relations. Compare relation
-	is used for the differential expression and group relations are used for the expressionset.
+	Schema showing a compare relation and three group relations. The compare relation
+	is used for differential expression, and group relations are used for expressionset.
 
 The simplest way to create relations is to download a relations template from a collection,
-define relations and finally apply it to the server by importing it. We will now download the
+define the relation, and finally apply it to the server by importing it. We will now download the
```
Comment (jvrakor) [inline docs/tutorial-diffexp.rst]: There is usually more than one relation in a collection.

File: docs/tutorial-diffexp.rst [hunk 2]
```diff
@@ -211,7 +211,7 @@ We can now check what we have created:
 We are now ready to run some expression and differential expression analyses.
 
 Cuffquant, Cuffnorm, and Cuffdiff
-================================
+=================================
```
Comment (jvrakor) [inline docs/tutorial-diffexp.rst]: This should not be a separate commit. When not sure you can think of it as: "Does this deserve a CHANGELOG entry?".

**Comment by jvrakor** [comment]
Try restructuring the commits so each commit corresponds to one of the files.

**Comment by jvrakor** [comment]
And add a CHANGELOG entry

File: docs/start.rst [hunk 3]
```diff
@@ -39,9 +39,9 @@ Registration
 
 Examples in documentation require access to a public Resolwe server.
 You can use a public Resolwe server `Genialis Platform`_ that is
-configured for the examples in tutorial. Some parts of the
+configured for the examples in this tutorial. Some parts of the
 documentation will work for registered users only. Please
-`create an account`_ on Genialis Platform before you continue,
+`create an account`_ on the Genialis Platform before you continue,
```
Comment (rafecooks) [inline docs/start.rst]: line 44: keep as was, omit article `the` before Genialis Platform

File: docs/start.rst [hunk 4]
```diff
@@ -4,15 +4,15 @@
 Getting started
 ===============
 
-This tutorial is for bioinformaticians. It will help you install the ReSDK
-and explain some basic commands. We will connect to a Resolwe server,
-do some basic queries, and align raw reads to a genome.
+This tutorial is for bioinformaticians. It will help you install the ReSDK and explain some basic
```
Comment (dblenkus) [inline docs/start.rst]: Please wrap lines at 80 characters. I know that some lines are longer, but they should be fixed when they are changed.

**Comment by mstajdohar** [comment]
What's with the tests, @dblenkus?

**Comment by tristanbrown** [comment]
@dblenkus

**Comment by mstajdohar** [comment]
@tristanbrown Is the code rebased to the master branch?

**Comment by tristanbrown** [comment]
@mstajdohar Yes, it is rebased to the master branch.

**Comment by mstajdohar** [comment]
@dblenkus, @tjanez This has been ready for quite some time.

**Comment by tjanez** [comment]
This is blocked by tests not passing. My WIP attempt at fixing them is at #137.

**Comment by tristanbrown** [comment]
@tjanez @dblenkus Some tests are still failing.

**Comment by tristanbrown** [comment]
The tests should work after #141 is merged.

**Comment by mstajdohar** [comment]
#141 was merged. Thanks for fixing the tests. Please rebase.

**Comment by tristanbrown** [comment]
Rebased.

**Comment by jkokosar** [comment]
@mstajdohar I believe this is ready to be merged
